packages/shared/lib/core/wallet/enums/asset-standard.enum.ts

```diff
@@ -0,0 +1,3 @@
+export enum AssetStandard {
+    IRC30 = 'IRC30',
+}
```

Comment:
This file can be deleted since it was moved / renamed to `token-standard.enum.ts` :+1: